extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -165,46 +157,39 @@ describe('amp-a4a', () => {
     });
 
     it('for SafeFrame rendering case', () => {
-      verifyNonAMPRender(a4a);
       // Make sure there's no signature, so that we go down the 3p iframe path.
       delete headers[SIGNATURE_HEADER];
       // If rendering type is safeframe, we SHOULD attach a SafeFrame.
       headers[RENDERING_TYPE_HEADER] = 'safeframe';
       fixture.doc.body.appendChild(a4aElement);
+      const onAmpCreativeRenderSpy = sandbox.spy(a4a, 'onAmpCreativeRender');
```

Comment:
This looks common to many test cases in this `.describe` block.  Can you move it to common `beforeEach` setup?